agasc/supplement/magnitudes/mag_estimate_report.py

```diff
@@ -271,6 +271,8 @@ def multi_star_html(
         ]
 
         agasc_stats = self.agasc_stats.copy()
+        if not np.all(np.diff(agasc_stats["agasc_id"]) > 0):
```

Comment:
This code does not need to run fast, so code simplicity should be prefered over speed. In other words, just sort the table and don't check first if it already sorted. These tables are small and sorting is fast anyway.

Reply:
ok, changed.